drivers/dma/dma_stm32_v1.c

```diff
@@ -410,44 +410,21 @@ bool stm32_dma_check_fifo_mburst(LL_DMA_InitTypeDef *DMAx)
 		case LL_DMA_MBURST_INC4:
 			return true;
 		case LL_DMA_MBURST_INC8:
-			if (fifo_level == LL_DMA_FIFOTHRESHOLD_1_2 ||
-			    fifo_level == LL_DMA_FIFOTHRESHOLD_FULL) {
-				return true;
-			} else {
-				return false;
-			}
+			return fifo_level == LL_DMA_FIFOTHRESHOLD_1_2 || fifo_level == LL_DMA_FIFOTHRESHOLD_FULL;
```

Comment:
how is this more readable? (and it's not fixing any issue so far)

Plus:
- you forgot the necessary () 
- and the lines are too long.